### PR TITLE
Add OpenTelemetry Node SDK docs

### PR DESCRIPTION
**Pull request by AbhiPrasad**
Mainly based on the README here: https://www.npmjs.com/package/@sentry/opentelemetry-node

Initial Context: https://github.com/getsentry/sentry/discussions/40712

Node SDK work: https://github.com/getsentry/sentry-javascript/issues/6000

### CODE REVIEW

**Comment by AbhiPrasad** [comment]
Moving this to draft because it may be blocked by @onurtemizkan's work in https://github.com/getsentry/sentry-docs/pull/5710.

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - unreal
+  - native.wasm
+description: "Using OpenTelemetry with Sentry Performance."
+---
+
+<Alert level="warning" title="Note">
+
+OpenTelemetry support for Sentry SDKs are in an alpha state. Features in alpha are still in-progress and may have bugs. We recognize the irony.
+
+</Alert>
```
Comment (imatwawana) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:37]:
```suggestion
<Include name="alpha-note.mdx" />
```

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+You can configure your [OpenTelemetry SDK](https://opentelemetry.io/) to send traces and spans to Sentry.
+
+## Installation
```
Comment (imatwawana) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:41]:
```suggestion
## Install
```

File: src/platform-includes/performance/opentelemetry-install/node.mdx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+```bash {tabTitle:yarn}
+npm install @sentry/node @sentry/opentelemetry-node
+```
+
+<Note>
+
+Note that @sentry/opentelemetry-node depends on the following peer dependencies:
+
+- @opentelemetry/api version 1.0.0 or greater
+- @opentelemetry/sdk-trace-base version 1.0.0 or greater, or a package that implements that, like @opentelemetry/sdk-node.
```
Comment (imatwawana) [inline src/platform-includes/performance/opentelemetry-install/node.mdx:14]:
```suggestion
- @opentelemetry/api, version 1.0.0 or greater
- @opentelemetry/sdk-trace-base, version 1.0.0 or greater, or a package that implements it, like @opentelemetry/sdk-node
```

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+<PlatformContent includePath="performance/opentelemetry-install" />
+
+## Usage
+
+<PlatformContent includePath="performance/opentelemetry-install" />
```
Comment (imatwawana) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:47]: Assuming this shouldn't be the same include file as above?
```suggestion
<PlatformContent includePath="performance/opentelemetry-setup" />
```

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
+---
+title: OpenTelemetry Support
```
Comment (imatwawana) [inline src/platforms/common/performance/instrumentation/opentelemetry.mdx:2]: You need to do a couple other things to get this page to be displayed I think. See [this PR](https://github.com/getsentry/sentry-docs/pull/5447/files#top) for help. Also, please add a dash in this file name to make it easier for humans to read later (i.e., open-telemetry). I'd definitely like to see this again once it's able to be previewed on Vercel.

**Review by imatwawana — COMMENTED** [review]
I've made a couple small edits, but this isn't yet able to be viewed in the preview. Some tweaks required.

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx (hunk 5 above)
Comment (mydea) [reply src/platforms/common/performance/instrumentation/opentelemetry.mdx:2]: The preview is working now: https://sentry-docs-git-abhi-otel-node-docs.sentry.dev/platforms/node/performance/instrumentation/opentelemetry/

Comment (mydea) [reply src/platforms/common/performance/instrumentation/opentelemetry.mdx:2]: Note that we always use `opentelemetry` without a dash, as it is `OpenTelemetry` originally.

**Review by mydea — APPROVED** [review]

File: src/platforms/common/performance/instrumentation/opentelemetry.mdx (hunk 5 above)
Comment (imatwawana) [reply src/platforms/common/performance/instrumentation/opentelemetry.mdx:2]: gotcha, okay

File: src/platform-includes/performance/opentelemetry-install/node.mdx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+```
+
+<Note>
+
+Note that @sentry/opentelemetry-node depends on the following peer dependencies:
+
+- @opentelemetry/api, version 1.0.0 or greater
+- @opentelemetry/sdk-trace-base, version 1.0.0 or greater, or a package that implements it, like @opentelemetry/sdk-node
+
+</Note>
```
Comment (imatwawana) [inline src/platform-includes/performance/opentelemetry-install/node.mdx:16]: Should the note about dependencies come before the install step?

File: src/platform-includes/performance/opentelemetry-setup/node.mdx [hunk 7]
```diff
@@ -0,0 +1,27 @@
+You need to register the SentrySpanProcessor and SentryPropagator with your OpenTelemetry installation:
```
Comment (imatwawana) [inline src/platform-includes/performance/opentelemetry-setup/node.mdx:1]:
```suggestion
You need to register `SentrySpanProcessor` and `SentryPropagator` with your OpenTelemetry installation:
```

**Review by imatwawana — APPROVED** [review]
Made a couple suggestions, but otherwise this looks good

File: src/platform-includes/performance/opentelemetry-install/node.mdx (hunk 6 above)
Comment (mydea) [reply src/platform-includes/performance/opentelemetry-install/node.mdx:16]: personally I find it better placed below (so the flow: this is how to install sentry. But make sure that you also have these dependencies installed). But we can of course still adjust this later as well! :)

**Pull request merged**